socorro/signature/siglists/prefix_signature_re.txt

```diff
@@ -45,6 +45,8 @@ dlmalloc
 dvm
 EtwEventEnabled
 extent_
+enum$<T>::unwrap
```

Comment:
These are used to build a regular expression so the $ here means "end of line". You need to escape them with a \.

```suggestion
enum\$<T>::unwrap
```